package.json

```
@@ -46,7 +46,7 @@
     "react-dom": "^18.0.0",
     "react-imask": "^6.4.2",
     "react-query": "^3.39.1",
-    "react-router-dom": "6",
+    "react-router-dom": "6.4.1",
```

Let's support latest minor version.
```suggestion
    "react-router-dom": "^6.4.1",
```